admin/controllers/pages/views/edit/view.php

```diff
@@ -345,6 +421,48 @@
 
 <script>
 
+	// preview widget
+	function preview_widget(el) {
+		fetch(<?php echo Config::uripath();?>'/admin/pages/edit/widget_preview/' + el.dataset.widgetid).then(function (response) {
```

Comment:
(response)=>{

Reply:
Change in https://github.com/HoltBosse/Alba/pull/434/commits/cb1854b5e8197b7abf23808e50fa8706c4874900